addon/components/mapbox-gl.js

```diff
@@ -96,6 +95,6 @@ export default Component.extend({
 
     this.mapLoaded(map);
 
-    set(this, 'map', map);
+    set(this, '_isMapLoaded', true);
```

Comment:
Should this be protected against destroyed component? 
```suggestion
	if (!this.isDestroyed && !this.isDestroying) {
	    set(this, 'map', map);
	    set(this, '_isMapLoaded', true);
	}
```

Reply:
we should be good with the destroyed check at the top of the function, no?

Reply:
Ooh sorry, the diff here hid the return :)